### PR TITLE
Cut 0.8.0

### PR DESCRIPTION
**Pull request by jkmassel**
Bump the version number in preparation for a new release

### CODE REVIEW

File: Automattic-Tracks-iOS/TracksConstants.m [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #import "TracksConstants.h"
 
 NSString *const TracksErrorDomain = @"TracksErrorDomain";
-NSString *const TracksLibraryVersion = @"0.7.1";
+NSString *const TracksLibraryVersion = @"0.8.0";
```
Comment (AliSoftware) [inline Automattic-Tracks-iOS/TracksConstants.m:4]: Wow was about to say you should update the `podspec` too but I just saw that it actually already [parses that `.m` file dynamically](https://github.com/Automattic/Automattic-Tracks-iOS/blob/develop/Automattic-Tracks-iOS.podspec#L3), nice. Not sure how stable that trick/solution is (e.g. what if we `pod trunk push` while the PWD is not the repo root, not sure the relative path used by `File.read` will work? etc) but something separate to a release cut, so might inspect that separately.

**Review by AliSoftware — CHANGES_REQUESTED** [review]
I think you need to run `pod install` in the demo project too, to [update its `Podfile.lock`](https://github.com/Automattic/Automattic-Tracks-iOS/blob/develop/TracksDemo/Podfile.lock) – as [the demo project's own `Podfile` references the lib by relative path (development pod)](https://github.com/Automattic/Automattic-Tracks-iOS/blob/develop/TracksDemo/Podfile#L7) so resolves against the local `podspec` which now (dynamically) defines the version to `0.8.0`.

File: Automattic-Tracks-iOS/TracksConstants.m (hunk 1 above)
Comment (jkmassel) [reply Automattic-Tracks-iOS/TracksConstants.m:4]: Yeah, I really don't know how I feel about the whole parsing-the-version thing in this repo 😅

**Review by AliSoftware — APPROVED** [review]
Looks good with the lockfile fix now 👍

**Comment by jkmassel** [comment]
> I think you need to run pod install in the demo project too, to update its Podfile.lock – as the demo project's own Podfile references the lib by relative path (development pod) so resolves against the local podspec which now (dynamically) defines the version to 0.8.0.

Yeah good call – I'll try to remember to get a linter rule going for this or something, else I'll for sure forget next time

File: Automattic-Tracks-iOS/TracksConstants.m (hunk 1 above)
Comment (AliSoftware) [reply Automattic-Tracks-iOS/TracksConstants.m:4]: I would not blame you if you decide to remove that parsing part and switch to hardcoded version before merging I you want to 😇

**Pull request merged**